### DISCUSSION TITLE
Interest in 2D vis for small molecules

### DISCUSSION THREAD

**Discussion by tulga-rdn** [MDAnalysis/mdanalysis#4469] [GSoC Discussions]
Hi all! I'm Tulga, a 3rd year bioengineering and chemistry student at KAIST. I have some experience with MDA and protein simulations through my work with Pogorelov lab at UIUC, as well as some RDKit and small molecules experience from my AI and chemistry courses. Looking forward to working on the 2D visualization of small molecules or workflow-based issues to make MDA better! More info about myself is on my LinkedIn: https://www.linkedin.com/in/tulga-erdene/

**Comment by jennaswa** [comment]
Hi Tulga, and welcome to MDAnalysis! We are finalizing our application process for this year's program and will be sharing a post on [our blog](https://www.mdanalysis.org/blog/) with all of the details very soon - stay tuned!
